iconic-tpd.php

```diff
@@ -9,6 +9,29 @@
  * Text Domain: iconic-tdp
  */
 
+/**
+ * Boot as an MU plugin.
+ */
+function iconic_tpd_boot() {
+	if ( ! function_exists( 'wp_get_current_user' ) ) {
+		include ABSPATH . 'wp-includes/pluggable.php';
+	}
+
+	// Boot only for administrators.
+	if ( ! current_user_can( 'administrator' ) || did_action( 'load-plugins.php' ) {
```

Comment:
This is executed at `muplugins_loaded` so it is impossible `did_action( 'plugins_loaded' )` will ever be true.

I've looked at core source code and `load-plugins.php` is triggered **after** `muplugins_loaded` so that one will also be always be.
🤢 we have to examine `$_SERVER` 😢

Reply:
Done.
Copied from the last PR.

Reply:
The other thing I was thinking was to load it on the plugins page, but add a note on the plugin to state that it's temporarily deactivated (and maybe disable the activate button)

Reply:
Or replace with an "enable" button for each disabled plugin. Just didn't want people to click activate when it's is actually already active.

Reply:
I'm no businessman so good design decisions may not come from me.

Reply:
I would admit that I have something to do with design if there would be no source code :)

Reply:
Maybe a yellow state...

![kép](https://user-images.githubusercontent.com/952007/109393602-06cf5780-7923-11eb-8edd-a883e3f608a3.png)

Reply:
Could work! We could always do that after...